code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -1182,11 +1299,19 @@
 								if (ckey(param) == ckey(A.name))
 									M = A
 									break
-						if (M)
 #ifdef TWITCH_BOT_ALLOWED
 							if (IS_TWITCH_CONTROLLED(M))
 								return
 #endif
+						else
+							var/list/target_list = src.get_targets(1, "mob") // Chrunb Erbrbt and Scales To Lizard highfive!
+							if(length(target_list))
+								M = tgui_input_list(src, "Pick someone to high-five!", "EmotiConsole v1.1.3", target_list, 0)
```

Comment:
i think having a timeout would be good for these, even if it's a whole minute

Reply:
Okay, set there to be a timeout of about 20 seconds.